samples/nrf9160/lwm2m_client/fota.rst

```diff
@@ -0,0 +1,299 @@
+.. code-block:: console
+
+   [INFO] fota.py - Upload mfw_nrf9160_update_from_1.3.4_to_1.3.4-FOTA-TEST.bin with instance 1 to Coiote
+   [INFO] fota.py - Binary mfw_nrf9160_update_from_1.3.4_to_1.3.4-FOTA-TEST.bin, Size 14472 (bytes)
+   [INFO] coiote.py - Creating fota resource for binary mfw_nrf9160_update_from_1.3.4_to_1.3.4-FOTA-TEST.bin with id lwm2m_client_fota_instance_1
+   [INFO] fota.py - Allocated Resource id lwm2m_client_fota_instance_1 for instance 1
+
+Fota update command
+-------------------
+
+For running the full FOTA process, firmware can be given as a resource ID that already exists on the server side, or as a binary file that will be uploaded automatically.
+
+To use a binary name:
+
++-----------------+--------------------------------+---------+--------------------------------------+
+| Python file     | Options                        | Command | Arguments with binary file name      |
++=================+================================+=========+======================================+
+| :file:`fota.py` | -id <device_id> -to <time_out> | update  | <instance_id> file  <binary_name>    |
++-----------------+--------------------------------+---------+--------------------------------------+
+
+When binary type is ``file``, script tries to discover given binary name from the :file:`samples/nrf9160/lwm2m_client/` file, or the :file:`samples/nrf9160/lwm2m_client/build/zephyr`.
+The ``update`` command uploads and generates binary resource IDs automatically when a binary file is used.
+The update default task timeout is 800 seconds.
+
+Following is an example of updating a modem instance by giving a binary file:
+
+.. code-block:: console
+
+   ./scripts/fota.py -id urn:imei:351358811331351 update 1 file mfw_nrf9160_update_from_1.3.4_to_1.3.4-FOTA-TEST.bin
+
+To use existing resource IDs:
+
++-----------------+-----------------+----------+------------------------------------------+
+| Python file     | Options         | Command  | Arguments with Resource ID               |
++=================+=================+==========+==========================================+
+| :file:`fota.py` | -id <device_id> | update   | <instance_id> resource  <resource_id>    |
++-----------------+-----------------+----------+------------------------------------------+
+
+Following is an example of uploading a binary and updating a modem by referring to the uploaded resource ID:
+
+.. code-block:: console
+
+   ./scripts/fota.py upload 1 mfw_nrf9160_update_from_1.3.4-FOTA-TEST_to_1.3.4.bin
+   ./scripts/fota.py -id urn:imei:351358811331351 update 1 resource lwm2m_client_fota_instance_1
+
+
+Testing advanced FOTA
+=====================
+
+Complete the following steps to test the advanced Fota firmware update with the lwm2m client sample and the :file:`/scripts/fota.py` file.
+
+
+   #. Download the released modem version 1.3.4 zip file from `nRF9160 DK Downloads`_.
+   #. Update the modem firmware using the nRF Programmer app of `nRF Connect for Desktop`_.
```

Comment:
Is it for the nRF Programmer app?
If so, 
The nRF nRF Connect for Desktop link (Website page, which is already mentioned here) has a description about nRF Programmer app.

The other option could be to link to the infocenter page: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fug_nc_programmer%2FUG%2Fnrf_connect_programmer%2Fncp_introduction.html 

I think it would be better to link to the infocenter page. If so, I can make this change in my commit.

Reply:
No, I meant to link to the instructions for updating the modem firmware on nRF9160-based devices.